_locales/fr/messages.json

```diff
@@ -61,69 +61,69 @@
         "message": "Minutes"
     },
     "Secs": {
-        "message": "Seconds"
+        "message": "Secondes"
     },
     "Hide": {
-        "message": "Ne Montre Jaimais"
+        "message": "Toujours cacher"
     },
     "hitGoBack": {
-        "message": "Clique retourne pour si vous avez manqué parti."
+        "message": "Revenir au début du segment"
```

Comment:
This message should be telling you to click the "unskip" button to return. This is the message telling you how to get back.

Reply:
A better translation would be "Cliquez ici pour revenir avant le saut du segment" (click here to go back before sponsor skip). I haven't seen this string in the UI so excuse my confusion.

Reply:
Here is where it appears. It is the message that appears after hitting the report button. The long one in the middle.

![](https://media.discordapp.net/attachments/605458918621315098/611314495297552388/new_notice.gif)

Reply:
Ok so if the button is "Revenir en arrière" this text would be "Cliquez sur Revenir en arrière pour revenir avant le saut du segment"